frb_example/with_flutter/macos/Podfile

```diff
@@ -1,4 +1,4 @@
-platform :osx, '10.11'
+platform :osx, '10.14'
```

Comment:
uber-nit: any reasons we upgrade this platform constraints? Would be great to keep maximal compatibility